libkpod/image/copy.go

```diff
@@ -100,8 +104,115 @@ func PushImage(srcName, destName string, options CopyOptions) error {
 	return nil
 }
 
+type imagePullStruct struct {
+	imageName   string
+	tag         string
+	registry    string
+	hasRegistry bool
+	transport   string
+}
+
+func (ips imagePullStruct) returnFQName() string {
+	fqname := fmt.Sprintf("%s%s/%s:%s", ips.transport, ips.registry, ips.imageName, ips.tag)
```

Comment:
nit, could you just do a return and skip the variable assignment?

Reply:
ok